src/TimeService.Business/Commands/LeaveTime/CreateLeaveTimeCommand.cs

```diff
@@ -47,8 +90,16 @@ public OperationResultResponse<Guid> Execute(CreateLeaveTimeRequest request)
 
             _validator.ValidateAndThrowCustom(request);
 
+            var existUsers = CheckUserExistence(userIds, errors);
```

Comment:
It is very strange logic here. You send one id, if response contains this, for what do you send to mapper userid in request and array with one element

Reply:
Move check user existence  to validator

Reply:
Fixed